src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -47,7 +47,11 @@ struct Lapse : db::SimpleTag {
   using type = Scalar<DataType>;
   static std::string name() noexcept { return "Lapse"; }
 };
-
+template <size_t Dim, typename Frame, typename DataType>
+struct DerivativesOfSpacetimeMetric : db::SimpleTag {
```

Comment:
Could you please add a short doxygen comment so this at least shows up in the documentation? Probably most of the tags in this file need that, but I'm fine with that being done in a separate PR. I've notated some of the classes with no dox in #1514

Reply:
added, squashed and rebased